lists/sa.csv

```diff
@@ -481,7 +481,6 @@ https://www.xlovecam.com/,PORN,Pornography,2019-05-02,Netalitica,
 http://www.xvidzz.com/,PORN,Pornography,2019-05-02,Netalitica,
 https://zbporn.com/,PORN,Pornography,2019-05-02,Netalitica,
 https://tiaz.site/,PORN,Pornography,2019-05-02,Netalitica,
```

Comment:
There has been multiple mentions of removing `https://tiaz.site/` in this PR but it missed in sa.csv